pkg/internal/cryptoutil/utils.go

```diff
@@ -8,8 +8,13 @@ package cryptoutil
 
 import (
 	"crypto"
+	"crypto/ed25519"
```

Comment:
suggest to add these changes to a separate file (legacy_utils.go) for easier decommissioning of code (if it ever happens in the future)

Reply:
It's done, only the nonce needed to be moved.